function.php

```diff
@@ -282,6 +255,7 @@ function getPolyhoodsByHood()
 		$rs = db::getInstance()->query("
 			SELECT polyhoods.polyid, lat, lon, hoodid
 			FROM polyhoods INNER JOIN polygons ON polyhoods.polyid = polygons.polyid
+			WHERE active=1
```

Comment:
Hier bräuchte ich jetzt z.B. deine Commit-Message, da hier ja die hoods Tabelle gar nicht vorkommt und ich jetzt wissen müsste, in welcher Tabelle du das noch eintragen willst (um zu überlegen, ob ich das gut finde).

Reply:
active Feld ist in Hoods und polyhoods. Wobei es in letzteren nicht notwendig wäre da der Flag in der Hoods ja schon gesetzt ist.

Reply:
Ach wild irgendwelche Felder habt ihr auch schon angelegt. Ich glaube nicht, dass das Feld in der Polyhood Tabelle Sinn macht. Genau wegen solcher Sachen will ich das eben nicht übers Knie brechen, weil sonst lauter Zeug gemacht wird, das nicht durchdacht ist.

Reply:
Diese Stelle gibts im neuen PR nicht mehr